src/js/data-model.jsx

```diff
@@ -1,82 +1,153 @@
-var React = require('react'),
-    _ = require('underscore');
+var React = require('react');
+var _ = require('underscore');
 
 
-var router = require('./router'),
-    facts = require('./facts');
+var CollapsibleText = require('./collapsible-text');
+var Dqa = require('./dqa');
+var Table = require('./table');
+var resources = require('./resources');
```

Comment:
Let's keep the convention of splitting imports by a comma rather than individual `var`declarations.

```javascript
var React = require('react'),
    _ = require('underscore'),
    ... ;
```